highfidelity/hifi
# MS20825: Login Screen Aspect/Crop with resizing

MS#[20825](https://highfidelity.manuscript.com/f/cases/20825/Add-cropping-and-scaling-to-the-background-image-on-the-login-screen)

# TEST PLAN
- Launch interface in desktop mode (ensure you are not logged in)
- See the login dialog pop up
- resize the window
- the background image of the login dialog should crop and the aspect ratio should be preserved
